{"name":"verifyNoPostsAvailableForInvalidUser","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"No response for Invalid User expected [null] but found [[]]","trace":"java.lang.AssertionError: No response for Invalid User expected [null] but found [[]]\r\n\tat org.testng.Assert.fail(Assert.java:99)\r\n\tat org.testng.Assert.failNotSame(Assert.java:1033)\r\n\tat org.testng.Assert.assertNull(Assert.java:965)\r\n\tat com.practicetest.apiTestAutomation.tests.TestIntegration.verifyNoPostsAvailableForInvalidUser(TestIntegration.java:72)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:564)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:764)\r\n\tat org.testng.TestRunner.run(TestRunner.java:585)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:384)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:286)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1140)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1069)\r\n\tat org.testng.TestNG.run(TestNG.java:1037)\r\n\tat org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)\r\n\tat org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)\r\n\tat org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)\r\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1628809136789,"stop":1628809138195,"uuid":"c534b5b7-4312-4177-98de-1d50cc19caf7","historyId":"dd68df4afdf972529500173f911a32b3","fullName":"com.practicetest.apiTestAutomation.tests.TestIntegration.verifyNoPostsAvailableForInvalidUser","labels":[{"name":"package","value":"com.practicetest.apiTestAutomation.tests.TestIntegration"},{"name":"testClass","value":"com.practicetest.apiTestAutomation.tests.TestIntegration"},{"name":"testMethod","value":"verifyNoPostsAvailableForInvalidUser"},{"name":"parentSuite","value":"Default suite"},{"name":"suite","value":"Default test"},{"name":"subSuite","value":"com.practicetest.apiTestAutomation.tests.TestIntegration"},{"name":"host","value":"LAPTOP-4265JSAQ"},{"name":"thread","value":"16896@LAPTOP-4265JSAQ.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[]}